verl/trainer/ppo/ray_trainer.py

```diff
@@ -460,7 +460,7 @@ def check_mutually_exclusive(mbs, mbs_per_gpu, name: str):
 
         # check multi_turn with tool config
         if config.actor_rollout_ref.rollout.multi_turn.enable:
-            assert config.actor_rollout_ref.rollout.multi_turn.tool_config_path is not None, "tool_config_path must be set when enabling multi_turn with tool, due to no role-playing support"
+            assert config.actor_rollout_ref.rollout.multi_turn.tool_config_path is not None or config.actor_rollout_ref.rollout.multi_turn.interaction_config_path is not None, "tool_config_path or interaction_config_path must be set when enabling multi_turn with tool, due to no role-playing support"
```

Comment:
As the number of fields under `multi_turn` is growing, I'd suggest create a dataclass for multi_turn config similar to   https://github.com/volcengine/verl/pull/2117 and perform the assertion (validation checks) in the `__post_init__` function of the dataclass. For unit test we prefer creating dataclass directly instead of resorting to `OmegaConf.create`

Reply:
I will open a issue to track this https://github.com/volcengine/verl/issues/2128